composer.json

```diff
@@ -25,6 +25,7 @@
         "laravel/slack-notification-channel": "2.3.0",
         "laravel/socialite": "5.1.3",
         "laravel/tinker": "2.5.0",
+        "lynndigital/select-or-custom": "dev-master",
```

Comment:
They really don't tag this huh 😬

Reply:
14 commits and no tags in the whole history of that repo